src/languages/en-US.ts

```diff
@@ -2250,6 +2257,37 @@ export default class extends Language {
 			extendedHelp: `This command shows a random starred message or the starboard usage and leaderboard for this server.`
 		}),
 
+		/**
+		 * ####################
+		 * SUGGESTIONS COMMANDS
+		 */
+
+		COMMAND_SUGGEST_DESCRIPTION: 'Posts a suggestion for the server.',
+		COMMAND_SUGGEST_EXTENDED: builder.display('suggest', {
+			extendedHelp: `Posts a suggestion to the server's suggestion channel, if configured.`,
+			explainedUsage: [
+				['suggestion', 'Your suggestion']
+			],
+			examples: [
+				"Let's make a music channel!"
+			],
+			reminder: 'You need to have setup a suggestions channel for this command to work. If you are an administrator, you will be given the chance to do so upon invoking the command.'
+		}),
+		COMMAND_RESOLVESUGGESTION_DESCRIPTION: "Set the suggestion's status.",
+		COMMAND_RESOLVESUGGESTION_EXTENDED: builder.display('resolvesuggestion', {
+			extendedHelp: `This command allows you to update a suggestion's status, marking it either as accepted, considered or denied.
+			It also can be configured to DM the author regarding the status of their suggestion, with the \`suggestions.on-action.dm\` setting.
+			Furthermore, in case you wish to preserve anonymity, you can hide your name using the \`suggestions.on-action\` setting, which can be overridden with the \`--hide-author\` and \`--show-author\` flags`,
+			examples: [
+				'1 accept Thank you for your suggestion!',
+				'1 a Thank you for your suggestion!',
+				'1 consider Hmm... we may do this, but it\'s really low priority',
+				'1 c Hmm... we may do this, but it\'s really low priority',
+				'1 deny There is no way this is going to happen.',
+				'1 d There is no way this is going to happen.'
+			]
```

Comment:
```suggestion
			examples: [
				'1 accept Thank you for your suggestion!',
				'1 a Thank you for your suggestion!',
				'1 consider Hmm... we may do this, but it\'s really low priority',
				'1 c Hmm... we may do this, but it\'s really low priority',
				'1 deny There is no way this is going to happen.',
				'1 d There is no way this is going to happen.'
			],
			reminder: `Suggestions also can be configured to DM the author regarding the status of their suggestion, with the \`suggestions.on-action.dm\` setting.
			Furthermore, in case you wish to preserve anonymity, you can hide your name using the \`suggestions.on-action\` setting, which can be overridden with the \`--hide-author\` and \`--show-author\` flags`
```

And the equivalent moving for spanish